index.md

```diff
@@ -62,7 +56,11 @@ Links to the online version of the SP 800-63 suite are below.
 </li>
 </ul>
 
-A set of [Frequently Asked Questions (FAQ) is available online](https://pages.nist.gov/800-63-FAQ/).
+#### Additional informative resources:
+* [Frequently Asked Questions (FAQ)](https://pages.nist.gov/800-63-FAQ/)
+* NIST SP 800-63-3 Implementation Resources ([PDF](https://www.nist.gov/document/sp-800-63-3-implementation-resources07012020pdf))
+* Conformance Criteria for SP 800-63A *Enrollment and Identity Proofing* ([PDF](https://www.nist.gov/document/conformance-criteria-sp-800-63a-enrollment-and-identity-proofing))
+* Conformance Criteria for SP 800-63B *Authentication and Lifecycle Management* ([PDF](https://www.nist.gov/document/conformance-criteria-sp-800-63b-authentication-and-lifecycle-management))
```

Comment:
I'm a little concerned about the size of the click target on these being only the letters 'PDF' but it's probably ok, and I can understand why it would be like this.

Reply:
The reason is that I'm planning on making the title of the implementation resources a link when we publish that on pages.nist.gov. I think people will notice that PDF is a link.